app/src/main/java/com/github/swent/echo/compose/authentication/CreateProfile.kt

```diff
@@ -134,6 +140,8 @@ fun ProfileCreationScreen(
 
     if (state == CreateProfileState.SAVED) {
         LaunchedEffect(state) { navAction.navigateTo(Routes.MAP) }
+    } else {
+        Log.d("CreateProfile", "State: $state")
```

Comment:
This log as well as all the others in this file should be removed or at least commented as they are debug logs